code/go/internal/validator/test/packages/missing_data_stream/manifest.yml

```diff
@@ -0,0 +1,27 @@
+format_version: 1.0.4
```

Comment:
@mtojek Cant' comment on the `.empty` commit but that looks like a hack :-) Should the README be required? @aleksmaus Is this because a `docs` directory is required?

Reply:
It's because the docs directory is open (doesn't have any limitation). We may enforce some rules, but this way we may block some user files. 

For reference: https://github.com/elastic/package-spec/blob/master/versions/1/spec.yml#L30

Reply:
I like the idea that it is enforce. And I think we required in the UI the README.md file, so we could enforce this one too?

Reply:
https://github.com/elastic/package-spec/pull/116